cli/gitconfig_types.mli

```diff
@@ -0,0 +1,5 @@
+type binding = string * string  (* Key/value pair *)
+
+type section = string * binding list (* Section name and contents *)
+
+type t = section list
```

Comment:
```suggestion
type sections = section list
```
Or `config`? I feel like `t` might not be clear for the reader...

Reply:
Sure, `config` seems nice.